api/controllers/stats.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const Trials = require('../models/trial');
+const Records = require('../models/record');
+const Locations = require('../models/location');
+const Promise = require('bluebird');
+
+function getStats(req, res) {
```

Comment:
Make sure you also catch any errors. If you don't do so, HapiJS will just hang waiting for a reply (which will eventually timeout). Check the other controllers to see how you could do it.